hack/install.sh

```diff
@@ -123,3 +139,4 @@ downloadZeabur "${latestVersion}" "${binary}"
 
 showZeaburHelp
 
+sudo mv zeabur /usr/local/bin
```

Comment:
Perhaps it would be more appropriate to include this command in the documentation? (Otherwise, users may find themselves prompted for a password and become confused.)

Reply:
it's very normal, all cli apps needs user's password to install cli app to `/usr/local/bin` path

Reply:
got it!